modules/ocf_mirrors/manifests/centos.pp

```diff
@@ -8,17 +8,17 @@
     recurse => true,
   }
 
-  cron {
+  ocf_mirrors::timer {
     'centos':
-      command => '/opt/mirrors/project/centos/sync-archive > /dev/null',
-      user    => 'mirrors',
-      hour    => '*/3',
-      minute  => '22';
+      exec_start => '/opt/mirrors/project/centos/sync-archive',
+      stdout     => 'null',
```

Comment:
I think we'd actually want to see the stdout get logged, given that this won't email us now. We'd also need to have some kind of health checks on all our mirrors (not just the ftpsync ones) so that we can tell when one has been failing for a while.